dupcop

```diff
@@ -19,19 +20,18 @@ const getVariants = (string) => ({
 });
 
 const substitute = (string, substitutionsPairs) => {
-  for (let pair of substitutionsPairs) {
-    const what = getVariants(pair[0]);
-    const to = getVariants(pair[1]);
+  for (const pair of substitutionsPairs) {
```

Comment:
If you like `const` you can use it in `for ... of` loops: new binding is created on every iteration, so no reassignments here.